backend/workflow_manager/workflow_v2/workflow_helper.py

```diff
@@ -358,8 +358,9 @@ def _update_pipeline_status(
                 f"with workflow execution: {workflow_execution}"
             )
 
-    @staticmethod
+    @classmethod
```

Comment:
@muhammad-ali-e why was this changed from a static method to class method? Could this have any other side effects? 
Couldn't we have done the `cls._set_result_acknowledge` at the end where this call is being used? 

No need to make any changes. Trying to understand the reason for this change.

Reply:
The method was changed to a class method to allow the use of cls for invoking `_set_result_acknowledge`.
 Moving c`ls._set_result_acknowledge` to where the method is called could be a possibility, but it would shift responsibility out of this method.